src/coreComponents/linearAlgebra/interfaces/hypre/mgrStrategies/CompositionalMultiphaseReservoirFVM.hpp

```diff
@@ -107,53 +107,34 @@ class CompositionalMultiphaseReservoirFVM : public MGRStrategyBase< 3 >
               HyprePrecWrapper & precond,
               HypreMGRData & mgrData )
   {
-    GEOS_LAI_CHECK_ERROR( HYPRE_MGRSetCpointsByPointMarkerArray( precond.ptr,
-                                                                 m_numBlocks, numLevels,
-                                                                 m_numLabels, m_ptrLabels,
-                                                                 mgrData.pointMarkers.data() ) );
-
-    GEOS_LAI_CHECK_ERROR( HYPRE_MGRSetLevelInterpType( precond.ptr, toUnderlyingPtr( m_levelInterpType ) ) );
-    GEOS_LAI_CHECK_ERROR( HYPRE_MGRSetLevelRestrictType( precond.ptr, toUnderlyingPtr( m_levelRestrictType ) ) );
-    GEOS_LAI_CHECK_ERROR( HYPRE_MGRSetCoarseGridMethod( precond.ptr, toUnderlyingPtr( m_levelCoarseGridMethod ) ) );
-    GEOS_LAI_CHECK_ERROR( HYPRE_MGRSetNonCpointsToFpoints( precond.ptr, 1 ));
-    GEOS_LAI_CHECK_ERROR( HYPRE_MGRSetLevelSmoothType( precond.ptr, m_levelSmoothType ) );
-    GEOS_LAI_CHECK_ERROR( HYPRE_MGRSetLevelSmoothIters( precond.ptr, m_levelSmoothIters ) );
-    GEOS_LAI_CHECK_ERROR( HYPRE_MGRSetTruncateCoarseGridThreshold( precond.ptr, 1e-20 )); // Low tolerance to remove only zeros
-
     // if the wells are shut, using Gaussian elimination as F-relaxation for the well block is an overkill
     // in that case, we just use Jacobi
     if( mgrParams.areWellsShut )
     {
       m_levelFRelaxType[0] = MGRFRelaxationType::jacobi;
     }
 
+    setReduction( precond, numLevels, mgrData );
+
+    // I assumed this is related to the fill in generated while eliminating
```

Comment:
It is related, however, it might be useful even after the recent fixes involving the volume constraint. We would need to evaluate with some tests... My intuition is that is will filter out some additional entries from the coarse grid operators but the impact will be very small since the input tolerance used here is so low (1e-20).

Reply:
OK, let's leave it for now and reevaluate after testing.

Reply:
Maybe we should remove all the calls to `HYPRE_MGRSetTruncateCoarseGridThreshold`> In PR #2499 (merged in `develop`), I actually removed all of them already after we realized that this was making the matrix structurally non-symmetric (which was confusing for hypre RCM)

Reply:
OK, I messed up while merging `develop` into my branch. I'll remove all calls. Thanks